Remotion/Web/Core/Res/HTML/DropDownMenu.js

```diff
@@ -286,7 +286,7 @@ function DropDownMenu_BeginOpenPopUp(menuID, context, evt)
     statusPopup.setAttribute('aria-labelledby', menuButton[0].id);
   _dropDownMenu_currentStatusPopup = statusPopup;
   $(statusPopup).iFrameShim({ top: '0px', left: '0px', width: '100%', height: '100%' });
-  document.body.appendChild(statusPopup);
+  document.getElementById(menuID).parentElement.appendChild(statusPopup);

```

Comment:
problem: context-menu for TreeView (e.g. SingleTestTreeView.aspx) also opens a popup menu. Parent element is a span, popup is a div, this this invalid HTML.

Reply:
True. I guess the dropdown menu does not need to be a `div` anyway right? So that would be one way to fix it. I could also walk up the DOM until an `UpdatePanel` is found (or the end is reached) and place the popup there.

Reply:
The dropdown element must be a div. There's an UL inside, so we have to nest it in a blocklevel element.

Correct about the walk. 

Two options:
a) walk up until you find a DIV or TD element (or root) and be done with it.

b) Update Panel IDs are available on async-Pages via {{Sys.WebForms.PageRequestManager.getInstance()._updatePanelIDs}}, which is a tad hacky since the _updatePanelIDs is not actually a public API. It probably won't matter, though. The problem with this approach though, is that the popup is a blocklevel-element, so if the updatepanel is a SPAN, it won't be correct html anyway. This means the behavior would be correct, but the html markup won't be. In theory, we could add a logstatement if we find this scenario and just print an error if the popup doesn't properly work with the currect page structure.

Reply:
Well a) sounds better imo. Will do.

Reply:
DIV / TD / TH